nautobot/dcim/tests/test_models.py

```diff
@@ -1121,3 +1121,20 @@ def test_power_panel_validation(self):
             'Rack group "Rack Group 1" belongs to a location ("Location 2") that does not contain "Location 1"',
             str(cm.exception),
         )
+
+
+class SiteTestCase(TestCase):
+    def test_latitude_or_longitude(self):
+        """Test latitude and longitude is parsed to sting"""
```

Comment:
```suggestion
        """Test latitude and longitude is parsed to string."""
```

Good to have a test case here, however, its his correct? The test values here are floats, and since this is a `Decimal` field, the more decimal places, the less precision there will be unless using `Decimal` objects, yes?

Reply:
Yes, but for a test case this should do right?